### PR TITLE
Availability Rules and Embed Tokens

### PR DESCRIPTION
**Pull request by adambird**
with pre-release support for Scheduling Conversations

### CODE REVIEW

File: lib/cronofy/client.rb [hunk 1]
```diff
@@ -1093,6 +1093,23 @@ def real_time_scheduling(args = {})
...
+    # reference.
+    #
+    # Returns an embed token
+    #
+    # Raises Cronofy::CredentialsMissingError if no credentials available.
+    # Raises Cronofy::AuthenticationFailureError if the access token is no
+    # longer valid.
+    # Raises Cronofy::TooManyRequestsError if the request exceeds the rate
+    # limits for the application.
+    def embed_token
```
Comment (gshutler) [inline lib/cronofy/client.rb]: These have never been released externally and won't be as UI Element tokens are the preferred path so support should be removed.

File: spec/lib/cronofy/client_spec.rb [hunk 2]
```diff
@@ -2690,4 +2690,145 @@
...
+        )
+    end
+
+    subject { client.lookup_scheduling_conversation(token) }
+
+    it_behaves_like 'a Cronofy request'
+    it_behaves_like 'a Cronofy request with mapped return value'
+  end
+
+  describe '#upsert_availability_rule' do
```
Comment (gshutler) [inline spec/lib/cronofy/client_spec.rb]: Should have specs for get and delete too.

File: lib/cronofy/client.rb (hunk 1 above)
Comment (adambird) [reply lib/cronofy/client.rb]: Removed

File: spec/lib/cronofy/client_spec.rb (hunk 2 above)
Comment (adambird) [reply spec/lib/cronofy/client_spec.rb]: Done

**Comment by adambird** [comment]
Travis build failure seems to be due to an update to `rvm` https://travis-ci.org/cronofy/cronofy-ruby/jobs/497166034

**Comment by adambird** [comment]
@gshutler would like to release this

**Pull request merged**